extensions/wikia/TemplateClassification/Permissions.class.php

```diff
@@ -15,8 +15,18 @@ class Permissions {
 	 * @return bool
 	 */
 	public function shouldDisplayEntryPoint( \User $user, \Title $title ) {
+		return $this->userCanChangeType( $user, $title )
+		       && $title->inNamespace( NS_TEMPLATE );
```

Comment:
I think that's example we may want to change as MediaWiki guidelines for PHP explicitly says opposite
https://www.mediawiki.org/wiki/Manual:Coding_conventions#Line_continuation
Let's leave this way until we clear this out. 
@adamkarminski is on it.